price-feeder/oracle/provider/huobi.go

```diff
@@ -267,7 +267,7 @@ func (p *HuobiProvider) setTickerPair(ticker HuobiTicker) {
 func (p *HuobiProvider) setCandlePair(candle HuobiCandle) {
 	p.mtx.Lock()
 	defer p.mtx.Unlock()
-	// huobi time period comes in seconds
+	// convert huobi timestamp seconds -> milliseconds
```

Comment:
One thing I forgot to point out in the previous PR(s), is that whenever we're converting durations/times, it's best to use the `time` package for clarity.

So i'd do this as: `candle.Tick.TimeStamp * int64(time.Millisecond)`

Reply:
Used the conversion factor like this : 
```golang
candle.Tick.TimeStamp = candle.Tick.TimeStamp * int64(time.Second/time.Millisecond)
```